### PR TITLE
Server to server and server to agent support

### PR DESCRIPTION
**Pull request by johntdyer**
Server to server and server to agent support, PR #12 should be merged first

### CODE REVIEW

**Comment by johntdyer** [comment]
#12 should be merged first then we can do this one

**Comment by johntdyer** [comment]
ok, I think this is good to go

File: thousandeyes/resource_agent_to_server.go [hunk 1]
```diff
@@ -232,63 +271,100 @@ func resourceAgentServerDelete(d *schema.ResourceData, m interface{}) error {
...
 	}
-	testID := httpTest.TestId
+	testID := agentToServer.TestID
 	d.SetId(strconv.Itoa(testID))
-	return resourceAgentServerRead(d, m)
+	return resourceAgentToServerRead(d, m)
 }
 
-func buildAgentServerStruct(d *schema.ResourceData) *thousandeyes.AgentServer {
+// func combinePortAndServer(d *schema.ResourceData) string {
```
[the remark this answers is not in the document]
Comment (johntdyer) [reply thousandeyes/resource_agent_to_server.go:287]: nothing, thats trash, I'll remove

**Comment by william20111** [comment]
12 is merged but there are conflicts

**Comment by johntdyer** [comment]
FML, merge conflicts in go.sum files suck......

**Comment by william20111** [comment]
> FML, merge conflicts in go.sum files suck......

Haha they are brutal.

**Pull request merged**